src/main/pluginsServer.ts

```diff
@@ -141,17 +141,19 @@ export async function startPluginsServer(
             void jobQueueConsumer?.resume()
         })
 
-        // use one extra connection for redis pubsub
-        pubSub = await createRedis(server)
-        await pubSub.subscribe(server.PLUGINS_RELOAD_PUBSUB_CHANNEL)
-        pubSub.on('message', async (channel: string, message) => {
-            if (channel === server!.PLUGINS_RELOAD_PUBSUB_CHANNEL) {
+        // use one extra Redis connection for pub-sub
+        pubSub = new PubSub(server, {
+            [server.PLUGINS_RELOAD_PUBSUB_CHANNEL]: async () => {
                 status.info('⚡', 'Reloading plugins!')
-
                 await piscina?.broadcastTask({ task: 'reloadPlugins' })
                 await scheduleControl?.reloadSchedule()
-            }
+            },
+            'reload-action': async (message) =>
+                await piscina?.broadcastTask({ task: 'reloadAction', args: { actionId: parseInt(message) } }),
+            'drop-action': async (message) =>
+                await piscina?.broadcastTask({ task: 'dropAction`', args: { actionId: parseInt(message) } }),
```

Comment:
There's some "`" here after "dropAction". Can you make sure this code path is also tested?